test/index.js

```diff
@@ -17,6 +17,8 @@ var server = ssbServer({
   appKey: shsCap
 })
 
+// ssbServer() is sync function but does not actually guarantee to have a working server instance when it retruns :(
+setTimeout( function() { 
```

Comment:
I thought that this waiting period was handled by `ssbClient()`, but if you're getting bugs locally then maybe this doesn't hurt? I remember a post on SSB that this wasn't normal, but I could be confused.

Btw, small typo on the word "retruns", otherwise this LGTM.